pkg/e2e/version.go

```diff
@@ -37,6 +37,7 @@ func ChooseVersions() (err error) {
 		err = setupUpgradeVersion(clusterVersion, versionList)
 
 		if err != nil {
+			log.Printf("error while selecting upgrade version: %v", err)
```

Comment:
Do you need the extra log message here? Create should fail and report the error returned.

Reply:
ACK, will fix. (Need to rebase anyway)